flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,256 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *
+from future.moves.urllib.parse import urlparse
+
+import logging
+from collections import MutableSet
+
+import requests
+from requests import RequestException
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+
+log = logging.getLogger('ombi_list')
+
+def generate_title(item, strip):
```

Comment:
why is this not another static class method? seems inconsistant

Reply:
Dealing with a python noob who hasn't done any real coding in probably 10 years. Also another example of taking concepts directly from the trakt_list plugin.

Wasn't expecting to get high style points.. Figured there would be a lot to clean up.

Reply:
I hope you take this feedback in the spirit it's given, which is a scrutinising as possible 😄 It's best to be as nit-picky as possible at the time of the creation since maintenance of all of the 3rd party plugins is scarce. You did a great job and I honestly didn't feel that a python noob did this.

Reply:
In other words: We will probably be very rough, but you shouldn't take it personally, we just want high quality code.

Reply:
I appreciate the feedback.